tests/test_subtractTask.py

```diff
@@ -82,6 +93,59 @@ def test_equal_images(self):
                                                 makeStats(), statistic=afwMath.STDEV)
         self.assertFloatsAlmostEqual(differenceStd, np.sqrt(2)*noiseLevel, rtol=0.1)
 
+    def test_psf_size(self):
+        """Test that the image subtract task runs without failing, if
+        fwhmExposureBuffer and fwhmExposureGrid parameters are set.
+        """
+        noiseLevel = 1.
+        science, sources = makeTestImage(psfSize=2.4, noiseLevel=noiseLevel, noiseSeed=6)
+        template, _ = makeTestImage(psfSize=2.4, noiseLevel=noiseLevel, noiseSeed=7,
+                                    templateBorderSize=20, doApplyCalibration=True)
+
+        schema = afwTable.ExposureTable.makeMinimalSchema()
+        weightKey = schema.addField("weight", type="D", doc="Coadd weight")
+        exposureCatalog = afwTable.ExposureCatalog(schema)
+        kernel = measAlg.DoubleGaussianPsf(7, 7, 2.0).getKernel()
+        psf = measAlg.KernelPsf(kernel, template.getBBox().getCenter())
+
+        record = exposureCatalog.addNew()
+        record.setPsf(psf)
+        record.setWcs(template.wcs)
+        record.setD(weightKey, 1.0)
+        record.setBBox(template.getBBox())
+
+        customPsf = CustomCoaddPsf(exposureCatalog, template.wcs)
+
+        config = subtractImages.AlardLuptonSubtractTask.ConfigClass()
+        config.doSubtractBackground = False
+        task1 = subtractImages.AlardLuptonSubtractTask(config=config)
+
+        config = subtractImages.AlardLuptonSubtractTask.ConfigClass()
+        config.doSubtractBackground = False
+        config.makeKernel.kernel.active.fwhmExposureBuffer = None
+        config.makeKernel.kernel.active.fwhmExposureGrid = None
```

Comment:
Will need to be `config.makeKernel.fwhmExposureBuffer` if the parameters get moved.

Reply:
The way we have defined the `CustomCoaddPsf`, we cannot test that the task fails raising an `InvalidParameterError` since we are removing the support for these parameters to be `None`. The unit test will instead test explicitly that `evaluateMeanPsfFwhm` runs while `getPsfFwhm` throws an error.

Reply:
Now that we have a log message before `evaluateMeanPsfFwhm` is called after `getPsfFwhm` fails, the unit tests look for the log message. This essentially replaces the earlier version of the test by checking that the code path follows the lines in the `except` block.